tests/sweep_framework/architecture.py

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+def tt_smi_command(arch: str):
+    if arch == "grayskull":
+        return ["/home/software/syseng/gs/tt-smi", "-tr", "0"]
+    elif arch == "wormhole" or arch == "wormhole_b0":
```

Comment:
Get rid of `wormhole`

Reply:
Also the smi command is different on CI. It's called `tt-smi-metal`. You may want to consider that if you plan on running on the current CI infrastructure in the future.

Reply:
Fixed.

Reply:
Re CI: We'll come back to this when we figure out how we want to implement this in CI pipelines.